docs/guide/tutorials.rst

```diff
@@ -0,0 +1,244 @@
+	``IS_actual_concentration,concentration_units,dilution_factor``.
+	With a sample row that may look like the following :
+	``0p5ug,Serotonin,,0.5,1,ug/mL,1``
+
+#. traML.csv
+	This file contains a list of proteins with peptide names, the header 
+	follows the following format :
+	``ProteinName,FullPeptideName,transition_group_id,transition_name,``
+	``RetentionTime,Annotation,PrecursorMz,MS1 Res,ProductMz,MS2 Res,Dwell,``
+	``Fragmentor,Collision Energy,Cell Accelerator Voltage,LibraryIntensity,decoy,``
+	``PeptideSequence,LabelType,PrecursorCharge,FragmentCharge,FragmentType,``
+	``FragmentSeriesNumber,quantifying_transition,identifying_transition,``
+	``detecting_transition``.
+	With a sample entry that may look like the following :
+	``Serotonin,,Serotonin,Serotonin,1.87,112.2,0,``
+	``Unit,0,Unit,,,,,1,0,,Light,1,1,,1,TRUE,FALSE,TRUE``.
+
+#. workflow.csv
+	The workflow steps, which are parsed by SmartPeak to process the data, 
+	are listed in this file under the column ``command_name``. 
+	A full list of the commands can be found in :ref:`Workflow Commands`.
+
+
+Targeted quantitation with HPLC data
+------------------------------------
+
+This tutorial walks you through the workflow for analyzing targeted HPLC data
+ starting from input file generation, to processing the data in SmartPeak, 
+ to reviewing the data in SmartPeak, to reporting the results for later use.
+
+Objectives
+~~~~~~~~~~
+
+#. Obtaining the SOP for the workflow.
+#. Choosing a data set for demonstrating the workflow.
+#. Creating an optimized SmartPeak input templates for running the workflow.
+
+The Workflows include
+~~~~~~~~~~~~~~~~~~~~~
+
+#. Calculating the calibration curves using Standards
+#. Processing Unknowns
+
+Steps
+~~~~~
+
+The tutorial includes the following steps :
+
+#. Setting up the input files
+
+The data set used can be found in 
+`HPLC_UV_Standards <https://github.com/AutoFlowResearch/SmartPeak/tree/develop/src/examples/data/HPLC_UV_Standards>`_ and
+`HPLC_UV_Unknowns <https://github.com/AutoFlowResearch/SmartPeak/tree/develop/src/examples/data/HPLC_UV_Unknowns>`_ 
+for the HPLC UV Standards and HPLC UV Unknowns respectively.
+
+#. Defining the workflow in SmartPeak
+
+For HPLC UV Standards analysis, the following steps are saved 
+into the ``workflow.csv`` file. Alternatively, steps can be replaced, 
+added or deleted direclty from SmartPeakGUI. 
+A detailed explanation of each command step
+can be found in :ref:`Workflow Commands`.
+
+	* LOAD_RAW_DATA
+	* MAP_CHROMATOGRAMS
+	* EXTRACT_CHROMATOGRAM_WINDOWS
+	* ZERO_CHROMATOGRAM_BASELINE
+	* PICK_MRM_FEATURES
+	* CHECK_FEATURES
+	* SELECT_FEATURES
+	* CALCULATE_CALIBRATION
+	* STORE_QUANTITATION_METHODS
+	* QUANTIFY_FEATURES
+	* STORE_FEATURES
+
+The workflow steps for HPLC UV Unknowns are :
+
+	* LOAD_RAW_DATA
+	* MAP_CHROMATOGRAMS
+	* EXTRACT_CHROMATOGRAM_WINDOWS
+	* ZERO_CHROMATOGRAM_BASELINE
+	* PICK_MRM_FEATURES
+	* QUANTIFY_FEATURES
+	* CHECK_FEATURES
+	* SELECT_FEATURES
+	* STORE_FEATURES
```

Comment:
I would recommend adding some discussion on the data review.  Perhaps even mentioning the export methods you added recently?

Reply:
Could you please elaborate on which data to review?

Reply:
I was referring to the Chromatogram, Heatmap, Line, and Calibrators plots.  I think a brief description of what each of the plots show and what they can be used for would be sufficient here.

Reply:
Thanks, I have made the necessary updates to all sections above. The final walkthrough is ready and can be viewed here : https://smartpeak-.readthedocs.io/en/feat-aut783/guide/tutorials.html . Please let me know if we need to modify any of the sub sections before committing.